app/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+
+repositories {
+    mavenCentral()
+    maven { url 'http://download.crashlytics.com/maven' }
```

Comment:
Crashlytics is a part of Fabric now. If you were using Crashlytics before then you're using Fabric now, if you want it or not. You can have your internal discussion about switching to Fabric, but it would be more about 'should we use Fabric or not use Fabric?' since you basically have not other option.